src/styles/icons.css

```diff
@@ -24,51 +24,2397 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
-.icon-exit:before {
-  content: "\e905";
+.icon-Exit-Door:before {
+  content: "\e900";
+}
+.icon-Family-Large:before {
+  content: "\e901";
 }
-.icon-leaving:before {
+.icon-pound:before {
+  content: "\e902";
+}
+.icon-Police-Hat-Large:before {
   content: "\e903";
 }
-.icon-police:before {
+.icon-leaving-large:before {
   content: "\e904";
 }
-.icon-pound-note:before {
-  content: "\e900";
-}
-.icon-pound:before {
-  content: "\e901";
+.icon-pound-note-large:before {
+  content: "\e905";
 }
-.icon-police-bold:before {
+.icon-organisation:before {
   content: "\e906";
 }
-.icon-children:before {
-  content: "\e902";
+.icon-survey:before {
```

Comment:
Is everything below here intentionally included? Do we need all of it?

Reply:
I just thought it be better to have the icons there already if we want to offer different icons to charities? I can remove them if you don't agree

Reply:
I thought we decided to do one sheet/font per brand? Don't mind too much - but seems confusing to include a load of stuff we don't use.

Reply:
I've just loaded up the font files and checked - we definitely don't need all that. Please a set like we did for Haven that only includes the ones we are using for this version of the site.

Let me know if you need a hand on how to do that.